src/IntervalArith/Approx/Approx.purs

```diff
@@ -633,4 +707,23 @@ increasingPartialFunctionViaBounds :: (Approx -> Maybe Approx) -> (Approx -> May
 increasingPartialFunctionViaBounds f a = do
   resultL <- f $ lowerA a
   resultU <- f $ upperA a
-  pure $ resultL `unionA` resultU
\ No newline at end of file
+  pure $ resultL `unionA` resultU
+
+{-|
+Second argument is noise to be added to first argument. Used to allow for the
+error term when truncating a series.
+-}
+fudge :: Approx -> Approx -> Approx
```

Comment:
Is this a temporary function name? Can it be changed to something more descriptive?

Reply:
This comes directly from CDAR.  I prefer to keep it in sync with CDAR although some of the names are not very descriptive. Also, in this case it is hard to find a better name.  I think that fudge is a reasonable name for what it does - artificially lose precision.